src/apis.ts

```diff
@@ -1,38 +1,40 @@
 const isProduction = process.env.NODE_ENV === 'production';
 const mockPath = `${process.env.PUBLIC_URL}/__mocks__/apiRequests`;
 
+export const domain = window.location.hostname;
```

Comment:
```suggestion
const {
  location: { hostname: domain },
} = window;
```

Reply:
For explanation: export is not really required and object deconstruction is the preferred way of accessing nested object members in es6 (and upward)